### DISCUSSION TITLE
Welcome to chirp-seo Discussions!

### DISCUSSION THREAD

**Discussion by ryangittings** [Chirp-SEO/chirp-seo#1] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by techapplied** [comment]
I have been building and supporting websites with Perch / Perch Runway back to version 1. In recent years I have used Chirp SEO on a couple of sites and consider it an essential app for all but the smallest of sites. I hope Chirp and Perch continue to be developed and supported and the community continues to grow. 

I am a learn by example and persistant trial and error kind of web developer but happy to help with what I can.
